cpp/betweenness_centrality_module/algorithm/betweenness_centrality.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <stack>
```

Comment:
Organize imports such that custom-made libraries are divided from stl. See examples in other C++ parts of MAGE.